RecoEgamma/EgammaTools/interface/MVAValueMapProducer.h

```diff
@@ -115,7 +131,7 @@ void MVAValueMapProducer<ParticleType>::produce(edm::StreamID, edm::Event& iEven
     // Loop over particles
     for (auto const& cand : src->ptrs())
     {
-      std::vector<float> auxVariables = variableHelper_.getAuxVariables(cand, iEvent);
+      std::vector<float> auxVariables = variableHelper_.getAuxVariables(iEvent);
```

Comment:
Since this do not depend on ```cand``` any more, it could be taken out of the two nested loops: correct?